library/std/src/os/fd/raw.rs

```diff
@@ -12,11 +15,20 @@ use crate::os::unix::io::OwnedFd;
 #[cfg(target_os = "wasi")]
 use crate::os::wasi::io::OwnedFd;
 use crate::sys_common::{AsInner, IntoInner};
+#[cfg(target_os = "hermit")]
+use hermit_abi::{STDERR_FILENO, STDIN_FILENO, STDOUT_FILENO};
+#[cfg(not(target_os = "hermit"))]
+use libc::{STDERR_FILENO, STDIN_FILENO, STDOUT_FILENO};
```

Comment:
The CI issue happens when documenting std on Windows. `std::os::fd` exists even on Windows with `cfg(doc)`. We might work around like this:

```suggestion
#[cfg(target_os = "hermit")]
use hermit_abi as libc;
```

This, together with changing the `FILENO`s back to `libc::*_FILENO`, should solve the windows docs issue.